################################################################################
mission_director:
  ros__parameters:
    maneuver_mode: '1' # 1: Free running, 2: Docking, 3: DP, 4: Navigation

    # 1: Free running
      # 0: Speed mapping, 1: Turning, 2: Zigzag, 3: Pivot turn, 4: Crabbing, 5: Pull out, 6: Spiral
    # 2: Docking
      # 0: Heuristic (Entering), 1: Deep-RL (Entering), 2: Heuristic (Multi-stage), 3 : Deep-RL (Multi-stage)
    # 3: DP
      # TBD
    # 4: Navigation
      # TBD
    sub_maneuver_mode: '0'

    subsub_maneuver_mode: '0' # 0: Approaching stage, 1: Turning stage, 2: Entering stage, 3: Pushing stage
    sensor_mode: '0' # 0: Qualisys, 1: SLAM, 2: GPS-RTK, 3: Marker
    motor_mode: '0' # 0: Manual, 1: Auto
    status: '0' # Initializing, 1: Running, 2: Stop, 3: Standby, 4: Complete
################################################################################

